db/migrate/20250218125653_add_creator_id_to_problems.rb

```
@@ -1,5 +1,6 @@
 class AddCreatorIdToProblems < ActiveRecord::Migration[7.1]
   def change
-    # add_column :problems, :creator_id, :integer
-  end
-end
+    unless column_exists? :problems, :creator_id
+      add_column :problems, :creator_id, :integer
+    end
+  end
```

RuboCop (1.69.1)

[fatal] unexpected token $end
(Using Ruby 3.3 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)